pkg/virt-api/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -919,6 +920,18 @@ func ValidateVirtualMachineInstanceSpec(field *k8sfield.Path, spec *v1.VirtualMa
 		}
 	}
 
+	if (spec.Domain.Devices.BlockMultiQueue != nil) && (*spec.Domain.Devices.BlockMultiQueue == true) {
+		_, requestOk := spec.Domain.Resources.Requests[k8sv1.ResourceCPU]
+		_, limitOK := spec.Domain.Resources.Limits[k8sv1.ResourceCPU]
+		if (requestOk == false) && (limitOK == false) {
+			causes = append(causes, metav1.StatusCause{
+				Type:    metav1.CauseTypeFieldValueInvalid,
+				Message: fmt.Sprintf("MultiQueue for block devices can't be used without specifying CPU's."),
```

Comment:
proably better to explicitly include something like `without setting CPU requests and limits`.